src/EpiceaBrowsers-Tests/EpHasImpactFilterTest.class.st

```diff
@@ -0,0 +1,119 @@
+Class {
+	#name : #EpHasImpactFilterTest,
+	#superclass : #TestCase,
+	#instVars : [
+		'classFactory',
+		'impactFilter'
+	],
+	#category : #'EpiceaBrowsers-Tests-Filters'
+}
+
+{ #category : #tests }
+EpHasImpactFilterTest >> setUp [
+	classFactory := ClassFactoryForTestCase new.
+	impactFilter := EpImpactCodeChangeFilter new
+		environment: self class environment;
+		yourself.
+
+]
+
+{ #category : #tests }
+EpHasImpactFilterTest >> tearDown [
+	classFactory cleanUp
```

Comment:
TearDown should call tearDown at the end

Reply:
done